package.json

```diff
@@ -42,6 +42,18 @@
     }
   },
   "contributes": {
+    "commands": [
+      {
+        "command": "diffviewer.showLineByLine",
+        "title": "Show diff line by line",
+        "icon": "$(mirror)"
+      },
+      {
+        "command": "diffviewer.showSideBySide",
+        "title": "Show diff side by side",
+        "icon": "$(mirror~disabled)"
```

Comment:
This icon gives the impression that the command is disabled and can't be clicked, which is not true.

WDYT about using `file` icon for line-by-line and `files` icon for side-by-side?

Reply:
I understand what you mean about disabled; I took inspiration from the whitespace icon in diff editor which uses this visual pattern. 

Anyway, the mirror icon clashes with the "split editor right" icon that vscode puts everywhere so the file/files icon looks better. I'll commit momentarily.